docs/data_onboarding/linux.md

```diff
@@ -25,9 +25,21 @@ Splunkbase Download:
 [https://splunkbase.splunk.com/app/5659/](https://splunkbase.splunk.com/app/5659/) 
 
 
-Refer to `A-TA-linux_inputs` App on [this GitHub Repo](https://github.com/CrossRealms/Cyences-Input-Apps) for inputs.conf reference.
+### Collect AuditD logs from Linux
+
+Auditd data is collected via **Linux Auditd Technology Add-On**
+
+Splunkbase Download: 
+[https://splunkbase.splunk.com/app/4232/](https://splunkbase.splunk.com/app/4232/) 
```

Comment:
This App is archived.

Is this added on the Product list as to show on dependency??